internal/pkg/template/templates/environment/partials/vpc-resources.yml

```diff
@@ -12,6 +12,8 @@ VPC:
         Value: !Sub 'copilot-${AppName}-${EnvironmentName}'
 
 PublicRouteTable:
+  Metadata:
+    'aws:copilot:description': 'A custom Route Table that controls the routing for this environment's public subnets'
```

Comment:
nit:
```suggestion
    'aws:copilot:description': 'A custom route table that controls the routing for this environment's public subnets'
```

Reply:
😂 It's named "PublicRouteTable" probably for legacy reasons (when we only had public subnets), it's actually used for all
```suggestion
    'aws:copilot:description': 'A route table to direct network traffic within your VPC'
```

Reply:
Discussed offline. The route table is still only for public subnets.